src/main/java/bio/terra/cli/command/shared/options/ControlledCloningInstructionsForUpdate.java

```diff
@@ -0,0 +1,25 @@
+package bio.terra.cli.command.shared.options;
+
+import bio.terra.workspace.model.CloningInstructionsEnum;
+import picocli.CommandLine;
+
+public class ControlledCloningInstructionsForUpdate {
+  /**
+   * Use this --cloning flog for controlled resources.
+   *
+   * <p>This class is meant to be used as a @CommandLine.Mixin.
+   */
+
+  // Can't have a default. This is used for update. Need `null` in case user does not want to
+  // update cloning instructions.
+
+  @CommandLine.Option(
+      names = "--cloning",
+      description =
+          "Instructions for handling when cloning the workspace: ${COMPLETION-CANDIDATES}.")
```

Comment:
Are all enums valid? If not, can you replace `${COMPLETION-CANDIDATES}` with valid enums?

Reply:
Jay says REFERENCED is not valid for controlled resources. You can look here to see what's valid:

https://github.com/DataBiosphere/terra-workspace-manager/blob/main/service/src/main/java/bio/terra/workspace/service/resource/ResourceValidationUtils.java#L442